app/src/main/java/com/example/musictoned/workoutcreation/Workout.kt

```diff
@@ -10,6 +10,38 @@ data class Workout (var name: String, val exercises: LinkedList<WorkoutExercise>
        exercises.remove(exercise)
     }
 
+    fun getWorkoutDurationHourFormat(): String{
```

Comment:
Rather than calculating this every time, could we not just calculate this once and save it to persistent storage with the routine?

Reply:
Yes, this is possible, it just requires a little more thought so I was going to make a separate issue as this pr already has a lot of changes. I got a little carried away...